app/controllers/users/phone_setup_controller.rb

```diff
@@ -97,11 +96,5 @@ def new_phone_form_params
         :recaptcha_mock_score,
       )
     end
-
-    def confirm_user_in_account_setup
```

Comment:
I don't think we can remove this entire check. It was added in #8265 because this controller doesn't have the same authentication requirements in terms of re-authentication. Is there a way to fix this check so it allows a user in account setup to access it, but prevents users that are no longer in account setup from using it?